src/shared/styles/global.styles.js

```diff
@@ -0,0 +1,22 @@
+import { createGlobalStyle } from 'styled-components';
+import { normalize } from 'styled-normalize';
+
+const GlobalStyle = createGlobalStyle`
+  ${normalize}
+console.log('imtheme', props.theme);
```

Comment:
I guess this won't work by the way. You would need to do this

```
${props => console.log(...)}
```

Reply:
Removed thank you!